src/test/scala/cromwell/engine/backend/CromwellRuntimeAttributeSpec.scala

```diff
@@ -137,7 +143,7 @@ class CromwellRuntimeAttributeSpec extends FlatSpec with Matchers with EitherVal
     val descriptor = WorkflowDescriptor(WorkflowId(UUID.randomUUID()), SampleWdl.WorkflowWithFullGooglyConfig.asWorkflowSources())
     val call = descriptor.namespace.workflow.callByName("googly_task").get
     val unsupported = CromwellRuntimeAttributes.unsupportedKeys(call.task.runtimeAttributes.attrs.keys, BackendType.LOCAL)
-    unsupported shouldEqual Some("Found unsupported keys for backend 'LOCAL': cpu, disks, memory, zones")
+    unsupported shouldEqual Set("disks", "cpu", "zones", "memory")
```

Comment:
I'm not sure I understand the change to this test... why change from local -> jes?  Also shouldn't these keys be allowed for the JES backend?